docs/man/man1/melfagui.1

```diff
@@ -0,0 +1,54 @@
+.\" Copyright (c) 2023 andypugh
+.\"
+.\" This is free documentation; you can redistribute it and/or
+.\" modify it under the terms of the GNU General Public License as
+.\" published by the Free Software Foundation; either version 2 of
+.\" the License, or (at your option) any later version.
+.\"
+.\" The GNU General Public License's references to "object code"
+.\" and "executables" are to be interpreted as the output of any
+.\" document formatting or typesetting system, including
+.\" intermediate and printed output.
+.\"
+.\" This manual is distributed in the hope that it will be useful,
+.\" but WITHOUT ANY WARRANTY; without even the implied warranty of
+.\" MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.\" GNU General Public License for more details.
+.\"
+.\" You should have received a copy of the GNU General Public
+.\" License along with this manual; if not, write to the Free
+.\" Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301,
+.\" USA.
+.\"
+.\"
+.\"
+.TH melfagui "1"  "2023-07-13" "LinuxCNC Documentation" "The Enhanced Machine Controller"
+.SH NAME
+melfagui \- Vismach Virtual Machine GUI
+
+.SH DESCRIPTION
+\fBmelfagui\fR is one of the sample \fBVismach\fR GUIs for LinuxCNC,
+simulating a Mitsubishi serial manipulator
+
+See the main LinuxCNC documentation for more details.
+
+http://linuxcnc.org/docs/html/gui/vismach.html
+.
+.SH "SEE ALSO"
+\fBLinuxCNC(1)\fR
```

Comment:
The manual page file name is 'linuxcnc' with lower case letters, so I recommend changing these references to linuxcnc(1).